docker-compose.yml

```diff
@@ -92,7 +95,6 @@ services:
       GATEWAY_SERVER_URL: http://aerie_gateway:9000
       HASURA_CLIENT_URL: http://localhost:8080/v1/graphql
       HASURA_SERVER_URL: http://hasura:8080/v1/graphql
-      NODE_TLS_REJECT_UNAUTHORIZED: "0"
```

Comment:
We should keep this since this is the development docker-compose file

Reply:
@camargo this was fixed when I rebased and updated this PR. We can probably resolve this